ui/apps/platform/src/Containers/Policies/policies.utils.test.ts

```diff
@@ -0,0 +1,536 @@
+    describe('getClientWizardPolicy', () => {
+        test('should return a client policy object from a server policy object', () => {
+            const serverPolicy: Policy = {
+                id: 'e73359bd-68d0-48d6-8e3c-f81cf85e2574',
+                name: 'Test policy',
+                description: 'a description',
+                rationale: 'Rationale here',
+                remediation: 'Guidance here',
+                disabled: false,
+                categories: ['Cryptocurrency Mining', 'System Modification'],
+                lifecycleStages: ['BUILD', 'DEPLOY'],
+                eventSource: 'NOT_APPLICABLE',
+                exclusions: [
+                    {
+                        name: '',
+                        deployment: {
+                            name: 'archlinux',
+                            scope: {
+                                cluster: '5c5c9aae-9c92-4648-88a2-9e593c225fa1',
+                                namespace: 'kube-*',
+                                label: { key: 'app', value: 'archlinux' },
+                            },
+                        },
+                        expiration: null,
+                    },
+                    {
+                        name: '',
+                        image: { name: 'docker.io/library/archlinux:latest' },
+                        expiration: null,
+                    },
+                    {
+                        name: '',
+                        image: { name: 'docker.io/library/ghost:latest' },
+                        expiration: null,
+                    },
+                ],
+                scope: [
+                    {
+                        cluster: '5c5c9aae-9c92-4648-88a2-9e593c225fa1',
+                        namespace: 'ui-testing',
+                        label: { key: 'app', value: 'include1' },
+                    },
+                    {
+                        cluster: '5c5c9aae-9c92-4648-88a2-9e593c225fa1',
+                        namespace: 'ui-testing2',
+                        label: { key: 'app', value: 'include2' },
+                    },
+                ],
+                severity: 'LOW_SEVERITY',
+                enforcementActions: [],
+                notifiers: ['10a830c7-dc0b-4d9e-9505-4ae3b72d6b50'],
+                lastUpdated: '2024-08-08T19:27:43.987955873Z',
+                SORTName: 'Test policy',
+                SORTLifecycleStage: 'BUILD,DEPLOY',
+                SORTEnforcement: false,
+                policyVersion: '1.1',
+                policySections: [
+                    {
+                        sectionName: 'Policy Section 1',
+                        policyGroups: [
+                            {
+                                fieldName: 'Dockerfile Line',
+                                booleanOperator: 'OR',
+                                negate: false,
+                                values: [{ value: 'ENV=ENV=myapp=test' }, { value: 'USER=root' }],
```

Comment:
The oddly formatted value here is a manifestation of https://issues.redhat.com/browse/ROX-13221. Optimistically, capturing it in this test will push us one step closer to fixing it. :)